test/fixtures/transaction-response.json

```diff
@@ -0,0 +1,89 @@
+{
+  "type": "payment",
+  "address": "r9cZA1mLK5R5Am25ArfXFmqgNwjZgnfk59",
+  "specification": {
+    "source": {
+      "address": "r9cZA1mLK5R5Am25ArfXFmqgNwjZgnfk59",
+      "amount": {
+        "currency": "XRP",
+        "value": "1.112209"
```

Comment:
Yeah, that argument never made sense to me. Compiled languages should be able to handle missing keys too. But this is a javascript API, so I don't think it matters much anyway. Also, after talking to Bob he seemed to agree that having schemas alleviates the need for showing everything all the time.